.github/actions/roundup_summary.py

```diff
@@ -0,0 +1,107 @@
+    without having to query every single issue and count them."""
+    data = {
+        "query": """
+        {
+            repository(owner: "python", name: "cpython") {
+                open: issues(states: OPEN) { totalCount }
+                closed: issues(states: CLOSED) { totalCount }
+            }
+        }
+        """
+    }
+    response = requests.post(GRAPHQL_ENDPOINT, json=data, headers={
+        "Authorization": f"Bearer {token}",
+        "accept": "application/vnd.github.v3+json"
+    })
+    repo = response.json()["data"]["repository"]
+    open = repo["open"]["totalCount"]
+    closed = repo["closed"]["totalCount"]
+    return open, closed
+
+def get_issues(filters: Iterable[str], token: str, all_: bool = True):
+    """return a list of results from the Github search API"""
+    # TODO: if there are more than 100 issues, we need to include pagination
+    # this doesn't occur very often, but it should still be included just incase.
+    data = {"query": f"""
+        {{
+            search(query:"{' '.join(filters)}" type: ISSUE first: 100)
+            {{
+                pageInfo {{ hasNextPage endCursor startCursor }}
+                nodes {{
+                    ... on Issue {{
+                        title number author {{ login }} closedAt createdAt
+                    }}
+                }}
+            }}
+        }}
+    """}
+    response = requests.post(GRAPHQL_ENDPOINT, json=data, headers={
+        "Authorization": f"Bearer {token}",
+        "accept": "application/vnd.github.v3+json"
+    })
+    return response.json()["data"]["search"]["nodes"]
+
+def send_report(payload: str, token: str) -> int:
+    """send the report using the Mailgun API"""
+    params = {
+        "from": "Cpython Issues <github@mg.python.org>",
+        "to": "",
+        "subject": "Summary of Python tracker Issues",
+        "template": "issue-tracker-template",
+        "o:tracking-clicks": "no",
+        "h:X-Mailgun-Variables": payload,
+    }
+    response = requests.post(
+        MAILGUN_ENDPOINT,
+        auth=("api", token),
+        json=params)
+    return response.status_code
+
+if __name__ == '__main__':
+    date_from = date.today() - timedelta(days=7)
+    github_token = os.environ.get("github_api_token")
+    mailgun_token = os.environ.get("mailgun_api_key")
+
+    total_open, total_closed = get_issue_counts(github_token)
+    closed = get_issues(("repo:python/cpython", f"closed:>{date_from}", "type:issue"), github_token)
+    opened = get_issues(("repo:python/cpython", "state:open", f"created:>{date_from}", "type:issue"), github_token)
+    most_discussed = get_issues(
+        ("repo:python/cpython", "state:open", "type:issue", "sort:comments"),
+        github_token,
+        False)
+    no_comments = get_issues(
+        ("repo:python/cpython", "state:open", "type:issue", "comments:0", "sort:updated"),
+        github_token,
+        False)
+
+    payload = {
```

Comment:
This payload appears to be missing the following keys to render the proposed template:

- `timespan`
- `total_diff`